src/Parlot/Fluent/Helpers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Globalization;
+
+namespace Parlot.Fluent;
+
+internal static class Helpers
+{
+    internal static NumberStyles ToNumberStyles(this NumberOptions numberOptions)
+    {
+        var numberStyles = NumberStyles.None;
+
+        if (numberOptions.HasFlag(NumberOptions.AllowLeadingSign))
```

Comment:
IIRC the `HasFlag` can be really slow on older runtimes, `(numberOptions & NumberOptions.AllowLeadingSign) != 0` would perform the same way on all platforms

Reply:
We should really benchmark it with different runtime. But still HasFlag is more readable and I vote for it

Reply:
I benchmarked on .NET 6, but IDK in older runtimes.
https://github.com/ncalc/ncalc/pull/196

Reply:
I had no idea, I read about it and since 5.0 it should be the same, minus special cases where the method is big, so I will revert for safety. I understand @Bykiev 's argument and I would think the same, but in the case of Parlot we need to get the perf benefits we can if it's not crazy.

Reply:
This is full framework problem which extends to Unity if I understand correctly. But should not be a problem for anything modern.